api/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>com.microsoft.gctoolkit</groupId>
         <artifactId>gctoolkit</artifactId>
-        <version>3.0.3-SNAPSHOT</version>
+        <version>3.0.3</version>
```

Comment:
should inherit version from parent?

Reply:
using ${revision} from mvn version 3.5.0+